grpc/server/cmake/external/grpc/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+cmake_minimum_required(VERSION 2.8)
+
+project(external-grpc)
+
+include(ExternalProject)
```

Comment:
Where do external projects end up? Are they rmoeved when you do `make clean`? Do we need something like `make distclean` to remove everything, so we can have `make clean` to only remove dronecore itself?

Reply:
The target is defined a few lines below:

    -DCMAKE_INSTALL_PREFIX=$ENV{DRONECORESERVER_CARES_BINARY_DIR}

So it goes in `build/default/external` in the end. I am not sure what could be a good way to achieve what you suggest, though.

Reply:
I tried to add `make distclean` and modify `make clean` to keep `build/default/external` (I kept this change in a commit that can be squashed later).

I am not sure where the `default` in `build/default` comes from, though. Moreover, I am not sure if my command is understandable and if it works on all platforms:

    clean:
        @find build/default/ -mindepth 1 -path build/default/external -prune -o -exec rm -rf {} +

Reply:
`default` was just for the "native" build, so not ios and not android.

Reply:
Ok so maybe it is fine for now, and we'll have to see how we adapt that when we compile `dronecore_server` for Android and iOS